MVC/app.js

```diff
@@ -1,16 +1,23 @@
-const {getTopics, getAPI} = require("./controller")
+const {getTopics, getAPI, getArticle} = require("./controller")
 const express = require("express")
 
 const app = express()
 
 app.get('/api/topics', getTopics)
 app.get('/api', getAPI)
+app.get('/api/articles/:article_id', getArticle)
 
-// Return error for all requests to invalid endpoints i.e. get/invalidEndpoint
+/** Error Handling Middleware */
+app.use((err, req, res, next) => {
```

Comment:
nicely done, I'd just maybe think about extracting it into its own file at some point